Userland/Libraries/LibGUI/TextEditor.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2018-2020, Andreas Kling <kling@serenityos.org>
+ * Copyright (c) 2018-2021, Andreas Kling <kling@serenityos.org>
```

Comment:
Please don't modify other people's copyright lines. Feel free to add your own instead :)